lib/perl/Genome/Sys.pm

```diff
@@ -1518,8 +1528,12 @@ sub shellcmd {
                 {   # POE sets a handler to ignore SIG{PIPE}, that makes the
                     # pipefail option useless.
                     local $SIG{PIPE} = 'DEFAULT';
-                    my @cmdline = ('bash', '-c', "$shellopts_part $cmd");
-                    exec(@cmdline)
+
+                    unless (@cmdline) {
+                        @cmdline = ('bash', '-c', "$shellopts_part $cmd");
+                    }
+
+                    exec { $cmdline[0] } (@cmdline)
```

Comment:
> I've changed the `exec` line to use the "indirect object" syntax to avoid possibly spawning a shell if the `@cmdline` is only a single item.

Or as the previous comment shows, falling back on splitting words and sending as shell command.